dataverse-webapp/src/main/webapp/file.xhtml

```diff
@@ -122,8 +122,8 @@
                                                 </li> 
                                             </ui:fragment>
                                             <li>
-                                                <h:outputLink value="/editdatafiles.xhtml?selectedFileIds=#{FilePage.fileMetadata.dataFile.id}&#38;datasetId=#{FilePage.fileMetadata.datasetVersion.dataset.id}&#38;mode=SINGLE&#38;version=#{FilePage.version}">
-                                                    <h:outputText id="editFile-Multi-Link" value="#{bundle['file.dataFilesTab.metadata.header']}"/>
+                                                <h:outputLink value="/editSingleFile.xhtml?selectedFileIds=#{FilePage.fileMetadata.dataFile.id}&#38;datasetId=#{FilePage.fileMetadata.datasetVersion.dataset.id}&#38;version=#{FilePage.version}">
```

Comment:
If the mode=SINGLE was removed in xhml, is it still needed in the bean?

Reply:
You're right. I forgot to clean `EditDatafilesPage`